spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -203,6 +204,8 @@ def __init__(self, parent=None, background_color=None):
         splitter.setStyleSheet(
             f"border-radius: {SpyderPalette.SIZE_BORDER_RADIUS}"
         )
+        self.splitter.setChildrenCollapsible(False)
+        self.splitter.splitterMoved.connect(self.on_splitter_moved)
```

Comment:
```suggestion
        self.splitter.splitterMoved.connect(self._on_splitter_moved)
```

Let's use a private method for this.